From 22239688052e4dee9b54959bd1a86f49fbfeb846 Mon Sep 17 00:00:00 2001
From: Jonathan Bell <jonathan@raspberrypi.com>
Date: Wed, 8 Nov 2023 16:10:13 +0000
Subject: [PATCH 0717/1085] dt-bindings: mmc: sdhci-of-dwcmhsc: Add Raspberry
 Pi RP1 support

The DWC MSHC controller on RP1 needs differentiating from the generic
version.

Signed-off-by: Jonathan Bell <jonathan@raspberrypi.com>
---
 .../devicetree/bindings/mmc/snps,dwcmshc-sdhci.yaml          | 5 +++++
 1 file changed, 5 insertions(+)

--- a/Documentation/devicetree/bindings/mmc/snps,dwcmshc-sdhci.yaml
+++ b/Documentation/devicetree/bindings/mmc/snps,dwcmshc-sdhci.yaml
@@ -16,6 +16,7 @@ allOf:
 properties:
   compatible:
     enum:
+      - raspberrypi,rp1-dwcmshc
       - rockchip,rk3568-dwcmshc
       - rockchip,rk3588-dwcmshc
       - snps,dwcmshc-sdhci
@@ -34,6 +35,8 @@ properties:
       - description: axi clock for rockchip specified
       - description: block clock for rockchip specified
       - description: timer clock for rockchip specified
+      - description: timeout clock for rp1 specified
+      - description: sdio clock generator for rp1 specified
 
 
   clock-names:
@@ -44,6 +47,8 @@ properties:
       - const: axi
       - const: block
       - const: timer
+      - const: timeout
+      - const: sdio
 
   resets:
     maxItems: 5
